components/chassis.py

```diff
@@ -61,7 +60,9 @@ def __init__(
         )
 
         self.encoder = encoder
-        self.encoder.setPosition(encoder_offset)
+
+        # change to encoder.configMagnetOffset(encoder_offset)
+        self.encoder.talon.configIntegratedSensorOffset(encoder_offset)
```

Comment:
Does this method actually exist on a TalonSRX? Surely this refers to the Talon FX integrated sensor.